app/controllers/recipes_controller.rb

```diff
@@ -1,7 +1,44 @@
 class RecipesController < ApplicationController
-  def index; end
+  before_action :set_user, expect: [:update]
+  def index
+    # @user=current_user
+    @recipes = @user.recipes
```

Comment:
Kindly change the @user.recipes for the current user recipes. Now that you've already implemented the current user method with devise you should use it to get the recipes, shouldn't you?